lib/module.d.ts

```diff
@@ -0,0 +1,6 @@
+export class audio
+{
+    unlink(): void;
+    list(): string[];
+    link(target: string, mode: "input" | "output"): boolean;
```

Comment:
lmao
```suggestion
    link(target: string, mode: "include" | "exclude"): boolean;
```

Reply:
This should stay include exclude as it describes the behavior more precisely.

Output isn't clear, and doesn't properly describe what's happening